portal/node-server/email-service/EmailClient.ts

```diff
@@ -0,0 +1,113 @@
+import { prisma } from 'sil.appbuilder.portal.common';
+import { RoleId } from 'sil.appbuilder.portal.common/prisma';
+import SparkPost, { Attachment } from 'sparkpost';
+import {
+  addProperties,
+  EmailLayoutTemplate,
+  NotificationTemplate,
+  NotificationWithLinkTemplate,
+  ScriptoriaLogoBase64
+} from './EmailTemplates.js';
+
+if (!process.env.VITE_SPARKPOST_API_KEY) {
```

Comment:
SparkPost is only used during development.  The production system uses AWS SES which is a low-cost option for sending email within AWS EC2 instance (costs us about $0.30/mo).

We will need to upgrade this to a provider model. In S1, we also had a "provider" that logged the contents to the console.